docs/reference/module-options.md

```diff
@@ -41,13 +41,42 @@ require('apostrophe')({
 });
 ```
 
+## Using module options
+
+Most module options in core Apostrophe, and official modules, are used automatically for specific purposes. No additional work is needed to use them for their original purposes other than configuring them, if needed. In some cases it is helpful to reference the options object in custom module code. You can do this in any of the module configuration function sections using the `self` argument as `self.options`.
```

Comment:
"Most module options in core Apostrophe, and official modules"

All modules in this umbrella are "official", so I think the distinction is core/extended, not core/official. Not splitting hairs, just stumbled over this sentence a few times.

Reply:
well not all extended modules are official. But I can say "official extension modules" or something.